contents/blog/posthog-vs-hotjar.md

```diff
@@ -0,0 +1,538 @@
+        </tr>
+        <tr>
+            <td><strong>Customer.io</strong></td>
+            <td className="text-center"><span className="text-red text-lg">✖</span></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td>Sync data between platforms</td>
+        </tr>
+        <tr>
+            <td><strong>Sentry</strong></td>
+            <td className="text-center"><span className="text-red text-lg">✖</span></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td>Ingest Sentry errors for analysis</td>
+        </tr>
+        <tr>
+            <td><strong>Segment</strong></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td>Ingest events from Segment</td>
+        </tr>
+    </tbody>
+</table>
+</div>
+
+> **Want more?** Visit the [PostHog the app directory](/apps) for a full list of supported integrations.
+
+## Privacy, compliance, and security
+
+Regulatory compliance can be a critical need for many teams, especially if they operate in financial or healthcare industries. Regulations such as HIPPA and GDPR can require teams to store data in certain locations, or to protect data in certain ways. 
+
+<div className="overflow-x-auto -mx-5 px-5">
+<table className="w-full mt-4" style="min-width: 400px;">
+    <thead>
+        <tr>
+            <td className="w-3/12"></td>
+            <td><strong>Hotjar</strong></td>
+            <td><strong>PostHog</strong></td>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td><strong>GDPR compliant</strong></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td>EU hosting or data residency available</td>
+        </tr>
+        <tr>
+            <td><strong>HIPAA compliant</strong></td>
+            <td className="text-center"><span className="text-red text-lg">✖</span></td>
+            <td className="text-center">Self-host only</td>
+            <td>Business Associate Agreement available</td>
+        </tr>
+        <tr>
+            <td><strong>Data anonymization</strong></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td>Remove personally identifiable information</td>
+        </tr>
+        <tr>
+            <td><strong>Self-hosting option</strong></td>
+            <td className="text-center"><span className="text-red text-lg">✖</span></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td>Self-host and keep complete control of your data</td>
+        </tr>
```

Comment:
We could add a point about cookie-less tracking here. Potentially also mention apps such as the Filter-Out app, which prevent unwanted data being ingested.

Reply:
Re cookie-less tracking: It breaks a whole bunch of session replay functionality, so I dont think it's worth including 
https://posthog.com/tutorials/cookieless-tracking